src/utils/sendMail.js

```
@@ -0,0 +1,29 @@
+const nodemailer = require('nodemailer');
+
+require('dotenv').config();
+
+// eslint-disable-next-line no-console
+console.log('SMTP HOST:', process.env.SMTP_HOST);
+
+const sendMail = async (to, subject, html) => {
+  const transporter = nodemailer.createTransport({
+    host: process.env.SMTP_HOST,
+    port: process.env.SMTP_PORT,
+    secure: false,
+    auth: {
+      user: process.env.SMTP_USER,
+      pass: process.env.SMTP_PASS,
+    },
+  });
+
+  const info = await transporter.sendMail({
```

Wrap `transporter.sendMail` in try/catch to provide clearer errors, or let errors bubble but document behavior. Right now an exception from sendMail will propagate; prefer to catch and either rethrow a well-formed error or return an object indicating failure so callers can respond appropriately.